html/browsers/sandboxing/resources/post-sandbox-state-to-parent.html

```diff
@@ -0,0 +1,12 @@
+<h1>No Scripts</h1>
+<script>
+try {
+  // Attempting to access the parent frame's document should throw an exception
+  // in a sandboxed frame without allow-same-origin.
+  window.parent.document.body;
+  window.parent.postMessage("Not sandboxed", "*");
+} catch (e) {
+  window.parent.postMessage("Sandboxed", "*");
+}
+document.querySelector('h1').innerText = "Allows scripts"
```

Comment:
Please use `textContent` instead of `innerText`?  It's much more interoperable in terms of behavior....

Though I'm not sure why this line is here.